lib/wash/method.rb

```diff
@@ -61,6 +63,8 @@ def self.method(name, &block)
     end
 
     method(:stream) do |entry, _|
+      $stdout.sync = true
+      $stderr.sync = true
```

Comment:
Stream doesn't stream to stderr, so why sync it?

Reply:
It shouldn't, but it could. But that makes sense, I'll remove it.

Reply:
Actually, streamer already flushes after every chunk, so we shouldn't need this at all.